src/Mvc/Mvc.Core/src/Infrastructure/OrderedEndpointsSequenceProvider.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading;
+
+namespace Microsoft.AspNetCore.Mvc.Infrastructure
+{
+    internal class OrderedEndpointsSequenceProvider
```

Comment:
@javiercn can you investigate these failures to perform the "correct" translation of the changes from the release/5.0-rc2 branch into the master branch? I assume I messed up the merge in some fashion.